Templates/AppProject.stencil

```diff
@@ -22,4 +22,4 @@ import ProjectDescriptionHelpers
 // Creates our project using a helper function defined in ProjectDescriptionHelpers
 let project = Project.app(name: "{{ name }}",
                           platform: .{{ platform }},
-                          additionalTargets: ["{{name}}Kit", "{{name}}UI"])
+                          additionalTargets: ["{{name }}Kit", "{{name}}UI"])
```

Comment:
Should not be `{{ name }}` or `{{name}}`. Why did you add only trailing space?

Reply:
yep, you are right. The space does not matter but it looks weird, so I'll delete it 🙂